README.md

```diff
@@ -25,20 +25,16 @@ Sit back, relax, and let the Error Tailor do all the work!
 Run `ng add @ngneat/error-tailor`. This command updates the `AppModule`, and adds the `ErrorTailorModule` dependency:
 
 ```ts
-const errors = {
-  required: error => `This field is required`,
-  minlength: ({ requiredLength, actualLength }) => `Expect ${requiredLength} but got ${actualLength}`,
-  invalidAddress: error => `Address isn't valid`
-};
-
 @NgModule({
   declarations: [AppComponent],
   imports: [
     ReactiveFormsModule,
     ErrorTailorModule.forRoot({
       errors: {
-        useValue() {
-          return errors;
+        useValue: {
```

Comment:
revert this, please.

Reply:
I mean the constant.

Reply:
About this I prefer the concise example with inline useValue, but if you want the const back I'll recreate it on top an maybe export it, because sometimes I had trouble with AOT referring in NgModule something that is not export... I don't know if Ivy totally solved it 🤔

Reply:
I want to prevent the horizontal scroll :)

Reply:
This can be easy with a line break in the arrow function and a pritter-ignore :wink:
but if you prefer const I'll do it

Reply:
Do it your way.